api/policies.md

```diff
@@ -0,0 +1,81 @@
+---
+api_name: timescaledb_experimental.policies
+excerpt: Get information about all policies set on continuous aggregates
+topics: [information, continuous aggregates]
+keywords: [continuous aggregates, information]
+tags: [policies, compress, data retention, refresh, jobs]
+api:
+  license: community
+  type: view
+  experimental: true
+---
+
+import Experimental from "versionContent/_partials/_experimental.mdx";
+
+<!-- vale Google.Headings = NO -->
+<!-- markdownlint-disable-next-line line-length -->
+## timescaledb_experimental.policies <tag type="community" content="Community" /><tag type="experimental" content="Experimental" />
+<!-- vale Google.Headings = YES -->
+
+The `policies` view provides information on all policies set on continuous
+aggregates.
+
+<highlight type="note">
+Only policies applying to continuous aggregates are shown in this view. Policies
+applying to regular hypertables or regular materialized views are not displayed.
```

Comment:
@RafiaSabih Is this true? This is what I got from reading the comments on the tests

Reply:
Yes.